### PR TITLE
Commerce Hub - Add a couple of GSFs

### PR DESCRIPTION
**Pull request by sinourain**
Description
-------------------------
Add:
physical_goods_indicator maps to physicalGoodsIndicator inside of transactionDetails scheme_reference_transaction_id maps to schemeReferenceTransactionId inside of storedCredentials

SER-501

Unit test
-------------------------
Finished in 29.475893 seconds.

5511 tests, 77405 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications 100% passed

Remote test
-------------------------
186.97 tests/s, 2626.04 assertions/s

Rubocop
-------------------------
Inspecting 760 files
760 files inspected, no offenses detected

### CODE REVIEW

File: test/remote/gateways/remote_commerce_hub_test.rb [hunk 1]
```diff
@@ -42,6 +42,14 @@ def test_successful_purchase
     assert_equal 'Approved', response.message
   end
 
+  def test_successful_purchase_whit_physical_goods_indicator
+    @options[:physical_goods_indicator] = true
+    response = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success response
+    assert_equal 'Approved', response.message
+    assert_equal @options[:physical_goods_indicator], response.params['transactionDetails']['physicalGoodsIndicator']
```
Comment (Heavyblade) [inline test/remote/gateways/remote_commerce_hub_test.rb:50]: 💬 Comment;

GIven that you are testing a boolean value you can just use `assert`:

```ruby
assert response.params['transactionDetails']['physicalGoodsIndicator']
```

File: test/remote/gateways/remote_commerce_hub_test.rb [hunk 2]
```diff
@@ -109,7 +117,7 @@ def test_successful_purchase_with_stored_credential_framework
   def test_failed_purchase
     response = @gateway.purchase(@amount, @declined_card, @options)
     assert_failure response
-    assert_equal 'Unable to assign card to brand: Invalid.', response.message
```
Comment (Heavyblade) [inline test/remote/gateways/remote_commerce_hub_test.rb:112]: ⚠️ Warning:

Seems that we need to assert with `assert_match` on this, seems that depending on credentials/features available this message has that final `.`

**Review by Heavyblade — APPROVED** [review]
@sinourain looks pretty much OK, leaved you a couple of comments

File: test/remote/gateways/remote_commerce_hub_test.rb (hunk 2 above)
Comment (sinourain) [reply test/remote/gateways/remote_commerce_hub_test.rb:112]: Done!

**Review by naashton — APPROVED** [review]
Nice work! Would you mind adding a changelog entry?

**Pull request merged**